GoogleCloudPlatform/avocano
# feat: checkout page

## Description

**To test changes locally:**

* Running to see the cart features: `AVOCANO_PURCHASE_MODE=cart npm run start`
   * This should reveal checkout tab and cart, as well as checkout page.
* Running to see the original features: `npm run start`

## Checklist
